### PR TITLE
Update XWiki

### PR DESCRIPTION
**Pull request by tmortagne**

### CODE REVIEW

File: library/xwiki [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
-Tags: 10, 10.11, 10-mysql-tomcat, 10.11-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 11, 11.0, 11-mysql-tomcat, 11.0-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 Architectures: amd64
-GitCommit: 929e25ced230a060e552714145c30cf12374ccfd
-Directory: 10/mysql-tomcat
+GitCommit: 17951cc427bde059e25eca752d31364e20db3ab9
+Directory: 11/mysql-tomcat
 
-Tags: 10-postgres-tomcat, 10.11-postgres-tomcat, stable-postgres-tomcat, stable-postgres
+Tags: 11-postgres-tomcat, 10.0-postgres-tomcat, stable-postgres-tomcat, stable-postgres
```
Comment (yosifkit) [inline library/xwiki]: `10.0-postgres-tomcat`? :eyes: :wink:

Comment (tmortagne) [reply library/xwiki]: Right, good catch. Too many stuff to change at once...

**Comment by tmortagne** [comment]
Actually don't apply this just yet, there is a problem in 11.0.1.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index d5c81d6..fbd00c2 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,16 +1,19 @@
-xwiki:9
-xwiki:9-mysql-tomcat
-xwiki:9-postgres-tomcat
-xwiki:9.11
-xwiki:9.11-postgres-tomcat
-xwiki:9.11.8
-xwiki:9.11.8-postgres-tomcat
 xwiki:10
 xwiki:10-mysql-tomcat
 xwiki:10-postgres-tomcat
 xwiki:10.11
-xwiki:10.11-mysql-tomcat
 xwiki:10.11-postgres-tomcat
+xwiki:10.11.2
+xwiki:10.11.2-postgres-tomcat
+xwiki:11
+xwiki:11-mysql-tomcat
+xwiki:11-postgres-tomcat
+xwiki:11.0
+xwiki:11.0-mysql-tomcat
+xwiki:11.0-postgres-tomcat
+xwiki:11.0.1
+xwiki:11.0.2-mysql-tomcat
+xwiki:11.0.2-postgres-tomcat
 xwiki:latest
 xwiki:lts
 xwiki:lts-mysql
diff --git a/xwiki_latest/Dockerfile b/xwiki_latest/Dockerfile
index 748555c..cbc7a5a 100644
--- a/xwiki_latest/Dockerfile
+++ b/xwiki_latest/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=10.11.1
+ENV XWIKI_VERSION=11.0.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 5bebaa5949b716105b9afc3eac0a0d50e2f4b1d2edaca28e7b9fed00bad24d68
+ENV XWIKI_DOWNLOAD_SHA256 1c18d2f56d3e722a767dcc8747b73cbc44fd68f6acd37da736fa0bc2c3a98350
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts-postgres/Dockerfile b/xwiki_lts-postgres/Dockerfile
index 032258e..3a32022 100644
--- a/xwiki_lts-postgres/Dockerfile
+++ b/xwiki_lts-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.11.8
+ENV XWIKI_VERSION=10.11.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 e7b6f563012349dad2051fa7bb2ba5a5c2e736d11fcc41363a6acde16a4c128c
+ENV XWIKI_DOWNLOAD_SHA256 d2a65cbd8f4aea8fb8feae7d5778343b8f6ca88003b4c11b67915fb856b432af
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts-postgres/xwiki/docker-entrypoint.sh b/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
index a3d0515..e57a15b 100755
--- a/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
+++ b/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
@@ -121,13 +121,6 @@ function configure() {
   safesed "replacecontainer" $DB_HOST /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" $DB_DATABASE /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
 
-  echo '  Using filesystem-based attachments...'
-  xwiki_set_cfg 'xwiki.store.attachment.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.versioning.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.recyclebin.content.hint' 'file'
-  echo '  Using filesystem-based deleted documents...'
-  xwiki_set_cfg 'xwiki.store.recyclebin.content.hint' 'file'
-
   echo '  Generating authentication validation and encryption keys...'
   xwiki_set_cfg 'xwiki.authentication.validationKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   xwiki_set_cfg 'xwiki.authentication.encryptionKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
diff --git a/xwiki_lts-postgres/xwiki/hibernate.cfg.xml b/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
index 7f0949c..d7ebe10 100644
--- a/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
+++ b/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
@@ -99,6 +99,7 @@
     <mapping resource="activitystream.hbm.xml"/>
     <mapping resource="instance.hbm.xml"/>
     <mapping resource="mailsender.hbm.xml"/>
+    <mapping resource="notification-filter-preferences.hbm.xml"/>
 
   </session-factory>
 </hibernate-configuration>
diff --git a/xwiki_lts/Dockerfile b/xwiki_lts/Dockerfile
index 9a8009c..b5dd082 100644
--- a/xwiki_lts/Dockerfile
+++ b/xwiki_lts/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.11.8
+ENV XWIKI_VERSION=10.11.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 e7b6f563012349dad2051fa7bb2ba5a5c2e736d11fcc41363a6acde16a4c128c
+ENV XWIKI_DOWNLOAD_SHA256 d2a65cbd8f4aea8fb8feae7d5778343b8f6ca88003b4c11b67915fb856b432af
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts/xwiki/docker-entrypoint.sh b/xwiki_lts/xwiki/docker-entrypoint.sh
index a3d0515..e57a15b 100755
--- a/xwiki_lts/xwiki/docker-entrypoint.sh
+++ b/xwiki_lts/xwiki/docker-entrypoint.sh
@@ -121,13 +121,6 @@ function configure() {
   safesed "replacecontainer" $DB_HOST /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" $DB_DATABASE /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
 
-  echo '  Using filesystem-based attachments...'
-  xwiki_set_cfg 'xwiki.store.attachment.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.versioning.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.recyclebin.content.hint' 'file'
-  echo '  Using filesystem-based deleted documents...'
-  xwiki_set_cfg 'xwiki.store.recyclebin.content.hint' 'file'
-
   echo '  Generating authentication validation and encryption keys...'
   xwiki_set_cfg 'xwiki.authentication.validationKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   xwiki_set_cfg 'xwiki.authentication.encryptionKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
diff --git a/xwiki_lts/xwiki/hibernate.cfg.xml b/xwiki_lts/xwiki/hibernate.cfg.xml
index 3b7ac9d..bd0af68 100644
--- a/xwiki_lts/xwiki/hibernate.cfg.xml
+++ b/xwiki_lts/xwiki/hibernate.cfg.xml
@@ -97,6 +97,7 @@
     <mapping resource="activitystream.hbm.xml"/>
     <mapping resource="instance.hbm.xml"/>
     <mapping resource="mailsender.hbm.xml"/>
+    <mapping resource="notification-filter-preferences.hbm.xml"/>
 
   </session-factory>
 </hibernate-configuration>
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index bd18c39..e6bab90 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=10.11.1
+ENV XWIKI_VERSION=11.0.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 5bebaa5949b716105b9afc3eac0a0d50e2f4b1d2edaca28e7b9fed00bad24d68
+ENV XWIKI_DOWNLOAD_SHA256 1c18d2f56d3e722a767dcc8747b73cbc44fd68f6acd37da736fa0bc2c3a98350
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
```

</details>

File: library/xwiki [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
-Tags: 9-postgres-tomcat, 9.11-postgres-tomcat, 9.11.8-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
+Tags: 10-postgres-tomcat, 10.11-postgres-tomcat, 10.11.2-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
 Architectures: amd64, arm64v8
-GitCommit: 1617db38480798d515bcef595397e43518150dde
-Directory: 9/postgres-tomcat
+GitCommit: 17951cc427bde059e25eca752d31364e20db3ab9
+Directory: 10/postgres-tomcat
 
-Tags: 10, 10.11, 10-mysql-tomcat, 10.11-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 11, 11.0, 11.0.1, 11-mysql-tomcat, 11.0-mysql-tomcat, 11.0.2-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
```
Comment (yosifkit) [inline library/xwiki]: `11.0.1` vs `11.0.2-mysql-tomcat` :wink:

You are welcome to learn from any of our `generate-stackbrew-library.sh` scripts to help you generate this library file, like [cassandra](https://github.com/docker-library/cassandra/blob/197a99a985c50eefcf71ad6189764e7e81934596/generate-stackbrew-library.sh). I could make an initial pass if you like.

**Comment by tmortagne** [comment]
Really not my day it seems. That should be all this time, promise.

File: library/xwiki (hunk 2 above)
Comment (tmortagne) [reply library/xwiki]: Thanks for the suggestion, we'll look into it.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index d5c81d6..33444d7 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,16 +1,19 @@
-xwiki:9
-xwiki:9-mysql-tomcat
-xwiki:9-postgres-tomcat
-xwiki:9.11
-xwiki:9.11-postgres-tomcat
-xwiki:9.11.8
-xwiki:9.11.8-postgres-tomcat
 xwiki:10
 xwiki:10-mysql-tomcat
 xwiki:10-postgres-tomcat
 xwiki:10.11
-xwiki:10.11-mysql-tomcat
 xwiki:10.11-postgres-tomcat
+xwiki:10.11.2
+xwiki:10.11.2-postgres-tomcat
+xwiki:11
+xwiki:11-mysql-tomcat
+xwiki:11-postgres-tomcat
+xwiki:11.0
+xwiki:11.0-mysql-tomcat
+xwiki:11.0-postgres-tomcat
+xwiki:11.0.3
+xwiki:11.0.3-mysql-tomcat
+xwiki:11.0.3-postgres-tomcat
 xwiki:latest
 xwiki:lts
 xwiki:lts-mysql
diff --git a/xwiki_latest/Dockerfile b/xwiki_latest/Dockerfile
index 748555c..9602127 100644
--- a/xwiki_latest/Dockerfile
+++ b/xwiki_latest/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=10.11.1
+ENV XWIKI_VERSION=11.0.3
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 5bebaa5949b716105b9afc3eac0a0d50e2f4b1d2edaca28e7b9fed00bad24d68
+ENV XWIKI_DOWNLOAD_SHA256 ac9b1b917ec86946c7bcdfc53ddc39a9011003d8e563dcd7a77b74cf47aeeadf
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts-postgres/Dockerfile b/xwiki_lts-postgres/Dockerfile
index 032258e..3a32022 100644
--- a/xwiki_lts-postgres/Dockerfile
+++ b/xwiki_lts-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.11.8
+ENV XWIKI_VERSION=10.11.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 e7b6f563012349dad2051fa7bb2ba5a5c2e736d11fcc41363a6acde16a4c128c
+ENV XWIKI_DOWNLOAD_SHA256 d2a65cbd8f4aea8fb8feae7d5778343b8f6ca88003b4c11b67915fb856b432af
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts-postgres/xwiki/docker-entrypoint.sh b/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
index a3d0515..e57a15b 100755
--- a/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
+++ b/xwiki_lts-postgres/xwiki/docker-entrypoint.sh
@@ -121,13 +121,6 @@ function configure() {
   safesed "replacecontainer" $DB_HOST /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" $DB_DATABASE /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
 
-  echo '  Using filesystem-based attachments...'
-  xwiki_set_cfg 'xwiki.store.attachment.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.versioning.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.recyclebin.content.hint' 'file'
-  echo '  Using filesystem-based deleted documents...'
-  xwiki_set_cfg 'xwiki.store.recyclebin.content.hint' 'file'
-
   echo '  Generating authentication validation and encryption keys...'
   xwiki_set_cfg 'xwiki.authentication.validationKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   xwiki_set_cfg 'xwiki.authentication.encryptionKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
diff --git a/xwiki_lts-postgres/xwiki/hibernate.cfg.xml b/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
index 7f0949c..d7ebe10 100644
--- a/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
+++ b/xwiki_lts-postgres/xwiki/hibernate.cfg.xml
@@ -99,6 +99,7 @@
     <mapping resource="activitystream.hbm.xml"/>
     <mapping resource="instance.hbm.xml"/>
     <mapping resource="mailsender.hbm.xml"/>
+    <mapping resource="notification-filter-preferences.hbm.xml"/>
 
   </session-factory>
 </hibernate-configuration>
diff --git a/xwiki_lts/Dockerfile b/xwiki_lts/Dockerfile
index 9a8009c..b5dd082 100644
--- a/xwiki_lts/Dockerfile
+++ b/xwiki_lts/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.11.8
+ENV XWIKI_VERSION=10.11.2
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 e7b6f563012349dad2051fa7bb2ba5a5c2e736d11fcc41363a6acde16a4c128c
+ENV XWIKI_DOWNLOAD_SHA256 d2a65cbd8f4aea8fb8feae7d5778343b8f6ca88003b4c11b67915fb856b432af
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_lts/xwiki/docker-entrypoint.sh b/xwiki_lts/xwiki/docker-entrypoint.sh
index a3d0515..e57a15b 100755
--- a/xwiki_lts/xwiki/docker-entrypoint.sh
+++ b/xwiki_lts/xwiki/docker-entrypoint.sh
@@ -121,13 +121,6 @@ function configure() {
   safesed "replacecontainer" $DB_HOST /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
   safesed "replacedatabase" $DB_DATABASE /usr/local/tomcat/webapps/ROOT/WEB-INF/hibernate.cfg.xml
 
-  echo '  Using filesystem-based attachments...'
-  xwiki_set_cfg 'xwiki.store.attachment.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.versioning.hint' 'file'
-  xwiki_set_cfg 'xwiki.store.attachment.recyclebin.content.hint' 'file'
-  echo '  Using filesystem-based deleted documents...'
-  xwiki_set_cfg 'xwiki.store.recyclebin.content.hint' 'file'
-
   echo '  Generating authentication validation and encryption keys...'
   xwiki_set_cfg 'xwiki.authentication.validationKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
   xwiki_set_cfg 'xwiki.authentication.encryptionKey' "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
diff --git a/xwiki_lts/xwiki/hibernate.cfg.xml b/xwiki_lts/xwiki/hibernate.cfg.xml
index 3b7ac9d..bd0af68 100644
--- a/xwiki_lts/xwiki/hibernate.cfg.xml
+++ b/xwiki_lts/xwiki/hibernate.cfg.xml
@@ -97,6 +97,7 @@
     <mapping resource="activitystream.hbm.xml"/>
     <mapping resource="instance.hbm.xml"/>
     <mapping resource="mailsender.hbm.xml"/>
+    <mapping resource="notification-filter-preferences.hbm.xml"/>
 
   </session-factory>
 </hibernate-configuration>
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index bd18c39..68b74e0 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=10.11.1
+ENV XWIKI_VERSION=11.0.3
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 5bebaa5949b716105b9afc3eac0a0d50e2f4b1d2edaca28e7b9fed00bad24d68
+ENV XWIKI_DOWNLOAD_SHA256 ac9b1b917ec86946c7bcdfc53ddc39a9011003d8e563dcd7a77b74cf47aeeadf
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
```

</details>

**Comment by yosifkit** [comment]
Build test of #5356; 392d11d7; `amd64` (`xwiki`):

```console

$ bashbrew build xwiki:10
Using bashbrew/cache:3aa47bbbffba6de8a56ba276c6cf22347596b1e156c6b204b9af7dcc2b583537 (xwiki:10)
Tagging xwiki:10
Tagging xwiki:10.11
Tagging xwiki:10.11.2
Tagging xwiki:10-mysql-tomcat
Tagging xwiki:mysql-tomcat
Tagging xwiki:lts-mysql-tomcat
Tagging xwiki:lts-mysql
Tagging xwiki:lts

$ test/run.sh xwiki:10
testing xwiki:10
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:10-postgres-tomcat
Using bashbrew/cache:984910c52f3117be9784d12e9e34d0dd86f944b783fa4d79af7607b97ce65ca3 (xwiki:10-postgres-tomcat)
Tagging xwiki:10-postgres-tomcat
Tagging xwiki:10.11-postgres-tomcat
Tagging xwiki:10.11.2-postgres-tomcat
Tagging xwiki:postgres-tomcat
Tagging xwiki:lts-postgres-tomcat
Tagging xwiki:lts-postgres

$ test/run.sh xwiki:10-postgres-tomcat
testing xwiki:10-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:11
Building bashbrew/cache:79cb67cdcd67d3ca0c2b6d2aac972f391a525b18293e7b3ad782fc7c58d47580 (xwiki:11)
Tagging xwiki:11
Tagging xwiki:11.0
Tagging xwiki:11.0.3
Tagging xwiki:11-mysql-tomcat
Tagging xwiki:11.0-mysql-tomcat
Tagging xwiki:11.0.3-mysql-tomcat
Tagging xwiki:stable-mysql-tomcat
Tagging xwiki:stable-mysql
Tagging xwiki:stable
Tagging xwiki:latest

$ test/run.sh xwiki:11
testing xwiki:11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:11-postgres-tomcat
Building bashbrew/cache:0200726b097a2eeec49849a7590819bad3c9fddf7984ba50cee1ca78b6f7353a (xwiki:11-postgres-tomcat)
Tagging xwiki:11-postgres-tomcat
Tagging xwiki:11.0-postgres-tomcat
Tagging xwiki:11.0.3-postgres-tomcat
Tagging xwiki:stable-postgres-tomcat
Tagging xwiki:stable-postgres

$ test/run.sh xwiki:11-postgres-tomcat
testing xwiki:11-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**